src/MongoDB/Monitoring.h

```diff
@@ -17,6 +17,8 @@
 #ifndef PHONGO_MONITORING_H
 #define PHONGO_MONITORING_H
 
+#include <php.h>
```

Comment:
`php.h` is where `PHP_FUNCTION()` is defined. This wasn't an error before as you generally have `php.h` included before this file, but I considered it good practice to have headers include their dependencies.